persistent-postgresql/Database/Persist/Postgresql.hs

```diff
@@ -974,7 +988,11 @@ getColumn getter tableName' [ PersistText columnName
         , cGenerated = fmap stripSuffixes generationExpression'
         , cDefaultConstraintName = Nothing
         , cMaxLen = Nothing
-        , cReference = fmap (\(a,b,c,d) -> ColumnReference a b (mkCascade c d)) ref
+        , -- The ColumnReference always has a non-null SchemaNameDB. The default schema name
+          -- in Postgres is "public", but Postgres doesn't know whether a table with
+          -- schema "public" was explicitly given that schema by the Persistent
+          -- app developer.
+          cReference = fmap (\(a,schemaName,c,d,e) -> ColumnReference a (Just schemaName) c (mkCascade d e)) ref
```

Comment:
Given that one variable has been renamed here, it might be worthwhile just to rename all of them. If someone looks at this code later without the context of this particular diff, it's going to look really weird that most of the variables are just assigned letters